LICENCE

```diff
@@ -0,0 +1,13 @@
+Copyright (c) 2015-2017 Shiftgig Inc.
```

Comment:
I like the practice of putting the license type as the first line of the `LICENSE` file (followed by a blank line).  At the very least, it provides easy reference (for us as well as anyone looking for examples).